src/test/resources/bulk/fms/03_phenotype_annotation/IF_01_infer_primary_allele_annotation_from_secondary_allele_annotation.json

```diff
@@ -0,0 +1,41 @@
+[
+    {
+      "objectId": "PATEST:Gene0001",
```

Comment:
This is a secondary gene annotation for a primary allele annotation - the file is correct, but the filename should reflect that